src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallCommandParser.cs

```diff
@@ -55,6 +55,8 @@ public static Command GetCommand()
             command.AddOption(ToolCommandRestorePassThroughOptions.NoCacheOption);
             command.AddOption(ToolCommandRestorePassThroughOptions.InteractiveRestoreOption);
             command.AddOption(VerbosityOption);
+            command.AddOption(CommonOptions.ArchitectureOption());
+            command.AddOption(CommonOptions.OperatingSystemOption());
```

Comment:
Tool install is probably a lot more complex and we may want to leave it for a separate PR.